cypress/pages/main.js

```diff
@@ -0,0 +1,12 @@
+export default class MainPage {
```

Comment:
the task to create page object based on js object - not on class.
could you update your code

Reply:
updated the code (based on js object) - homework 12